feature-libs/order/components/order-details/order-detail-items/order-consigned-entries/order-consigned-entries.component.ts

```diff
@@ -22,4 +22,12 @@ export class OrderConsignedEntriesComponent {
 
   readonly OrderOutlets = OrderOutlets;
   readonly CartOutlets = CartOutlets;
+
+  normalizeFormattedAddress(formattedAddress: string): string {
```

Comment:
this function is in many places. Can we move it to the `order/root/utils/order-card-utils`?